linux/rsuite.sh

```diff
@@ -0,0 +1,183 @@
+#!/usr/bin/env bash
+
+# A POSIX variable
+OPTIND=1         # Reset in case getopts has been used previously in the shell.
+
+# vim:ts=5:sw=5:expandtab
+# we have a spaces softtab, that ensures readability with other editors too
+
+[ -z "$BASH_VERSINFO" ] && printf "\n\033[1;35m Please make sure you're using \"bash\"! Bye...\033[m\n\n" >&2 && exit 245
+[ $(kill -l | grep -c SIG) -eq 0 ] && printf "\n\033[1;35m Please make sure you're calling me without leading \"sh\"! Bye...\033[m\n\n"  >&2 && exit 245
+
+# Initialize our own variables:
+output=false
+no_opt=false
+#######
+BASEDIR="$(dirname $0)/../"
+private_key="../tools/"
+ca_cert="./tools/"
+output_dir="$BASEDIR/output/"
+
+
+
+#######
+
+show_logo() {
+    echo "  _____                                 _____       _ _       
+ |  __ \                               / ____|     (_) |      
+ | |__) |_____   _____ _ __ ___  ___  | (___  _   _ _| |_ ___ 
+ |  _  // _ \ \ / / _ \ '__/ __|/ _ \  \___ \| | | | | __/ _ \ 
+ | | \ \  __/\ V /  __/ |  \__ \  __/  ____) | |_| | | ||  __/
+ |_|  \_\___| \_/ \___|_|  |___/\___| |_____/ \__,_|_|\__\___|
+                                                                                                                            "    
+}
+show_help() {
+     cat << EOF
+$0 <options>, where <options> are:
+
+     -h                        what you are looking at
+     -d <file.apk>,            disassemble the apk (convert to smali)
+     -l,                       shows the name of the applications already disassembled
+     -b <application_name>,    build the provided application as an apk (use with -o)
+     -n,                       optimize images during build
+     -x <file.apk>,            decompile the apk, and run luyten to verify the source code
+     -o <file.apk>,            the name of apk that gets written (use with -b)
```

Comment:
It's important that this text is the same as in the help. Check `-b` parameter for example.